test/test_socket.lua

```diff
@@ -59,6 +59,14 @@ context("Nakama socket", function()
 		assert_true(done)
 	end)
 
+	test("It should be able to disconnect", function()
+		local client = nakama.create_client(config())
+		local socket = client.create_socket()
+
+		socket.disconnect()
+		assert_true(true)
```

Comment:
This assert will always pass, is this intended? Isn't there a better way to assert if the sockets was indeed disconnected?

Like:

`assert_nil(socket.connection)`